onnx/defs/schema.cc

```diff
@@ -288,10 +313,27 @@ void OpSchema::Verify(const NodeProto& node) const {
           "Unrecognized attribute: ", name, " for operator ", node.op_type());
     }
 
-   // Type would be UNDEFINED if not set
+    // Type would be UNDEFINED if not set
     if (attr_proto.type() != expected_type) {
-      fail_check(
-          "Mismatched attribute type in '", node.name() + " : " + name, "'");
+      // Expected attribute type is "Tensor", in this case, "FLOAT", "FLOATS",
+      // "INT", "INTS", "STRING", "STRINGS" should be also OK.
```

Comment:
this is the only change, all others are format changes.

Reply:
I thought we would only touch `Constant`. Why do we add these lines?